internal/services/appservice/web_app_hybrid_connection_resource.go

```diff
@@ -231,12 +231,7 @@ func (r WebAppHybridConnectionResource) Read() sdk.ResourceFunc {
 				}
 
 				hybridConnectionsClient := metadata.Client.Relay.HybridConnectionsClient
-				hybridConnectionID, err := hybridconnections.ParseHybridConnectionID(appHybridConn.RelayId)
-				if err != nil {
-					return err
-				}
-
-				ruleID := hybridconnections.NewHybridConnectionAuthorizationRuleID(id.SubscriptionId, hybridConnectionID.ResourceGroupName, appHybridConn.ServiceBusNamespace, *existing.Name, appHybridConn.SendKeyName)
+				ruleID := hybridconnections.NewHybridConnectionAuthorizationRuleID(id.SubscriptionId, relayId.ResourceGroupName, appHybridConn.ServiceBusNamespace, *existing.Name, appHybridConn.SendKeyName)
```

Comment:
this is using a different resource group, which is going to be a breaking change - what's the reasoning for this?

Reply:
`hybridConnectionID` and `relayId` are both parsed from `appHybridConn.RelayId`, which we parsed twice duplicated from above, thus removing one of them. This is similar to https://github.com/hashicorp/terraform-provider-azurerm/pull/20237#discussion_r1093117062

Reply:
since we're querying the Hybrid Connection Rule, doesn't this want to be this?

```suggestion
				ruleID := hybridconnections.NewHybridConnectionAuthorizationRuleID(appHybridConn.SubscriptionId, appHybridConn.ResourceGroupName, appHybridConn.ServiceBusNamespace, *existing.Name, appHybridConn.SendKeyName)
```

Reply:
Right. I've updated subscriptionID for composing the Authorization Rule ID to be the one from appHybridConn, which is `relayId`